### PR TITLE
solution3

### PR DESCRIPTION
**Pull request by andriy-chornyy**
- [DEMO LINK](https://andriy-chornyy.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://andriy-chornyy.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

**Review by ArturErmolenko — CHANGES_REQUESTED** [review]
input doesn't inherit body font-face, that's why your tests didn't passed

**Comment by andriy-chornyy** [comment]
> input doesn't inherit body font-face, that's why your tests didn't passed

Я не знаю почему, и не могу исправить без четких указаний, я делаю такое первый раза в жизни

**Comment by ArturErmolenko** [comment]
> > input doesn't inherit body font-face, that's why your tests didn't passed
> 
> Я не знаю почему, и не могу исправить без четких указаний, я делаю такое первый раза в жизни

пробывал погуглить?

**Review by ArturErmolenko — CHANGES_REQUESTED** [review]
you need to add fonts on input placeholder

**Review by ArturErmolenko — CHANGES_REQUESTED** [review]
add this rule

input {
   font: inherit;
 }

**Comment by alexandra-protyanova** [comment]
Снимок экрана 2022-12-12 в 10 10 42
Снимок экрана 2022-12-12 в 10 10 50
According to mockup height of bars should be `70px` and `42px`

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
+@font-face {
+  font-family: Avenir, sans-serif;
```
Comment (alexandra-protyanova) [inline src/style.css:2]: `sans-serif` is odd here

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  margin-top: 20px;
+  position: relative;
+}
+
+.search-bar {
+  width: 100%;
+  font-family: Avenir, sans-serif;
+  color: #3d4e61;
+  font-weight: 800;
+  background: #fff;
```
Comment (alexandra-protyanova) [inline src/style.css:33]: white - is default color, remove it

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 1 above)
Comment (alexandra-protyanova) [reply src/style.css:2]: [CHECKLIST](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) point 7

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,95 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+
+}
```
Comment (alexandra-protyanova) [inline src/style.css:7]:
```suggestion
@font-face {
  font-family: Avenir;
  src: url(./fonts/Avenir-Book.ttf);
  font-style: normal;
  font-weight: 300;
}
```

Comment (alexandra-protyanova) [inline src/style.css:7]:
```suggestion
@font-face {
  font-family: Avenir;
  src: url(./fonts/Avenir-Book.ttf);
  font-style: normal;
  font-weight: 300;
}
```

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+@font-face {
```
Comment (alexandra-protyanova) [inline src/style.css:9]: here you imported font, let us set them to `html` selector and use `inherit` everywhere else

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+.search {
+  font: inherit;
+}
+
+.form1 {
+  height: 70px;
+}
+
+.form2 {
```
Comment (alexandra-protyanova) [inline src/style.css:93]: do not use numbers in class names

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]

**Review by alexandra-protyanova — APPROVED** [review]

**Pull request closed without merging**